### PR TITLE
Mobile view minor fixes and adding typing animation

### PR DESCRIPTION
**Pull request by aattsai**
The background image (I kept only NYC for mobile) seems fine on simulator. Let's see if it still zooms in on mobile after this merge.

### CODE REVIEW

File: css/mobile.css [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
-    .mission-statement {
-      position: relative;
-      font-size: 4.5em;
-      line-height: 1em;
-      padding-top: 33vh;
-      font-family: 'Didot', 'Comic Sans';
-      text-align: center;
-      color: #333436;
-
-    }
```
Comment (dijonkitchen) [inline css/mobile.css:20]: Loving this. We should just have one CSS file and it should work mobile-first!

File: css/mobile.css [hunk 2]
```diff
@@ -11,6 +11,18 @@
     padding-top: 18vh;
   }
   .logo {
-    margin-top: 0;
+    margin-top: 5em;
```
Comment (dijonkitchen) [inline css/mobile.css:14]: I always forget about rem vs em, but I think rem would be more universal last I looked. Check it out.

File: index.html [hunk 3]
```diff
@@ -9,6 +9,7 @@
   <link rel="stylesheet" type="text/css" href="css/mobile.css">
   <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.3/components/icon.min.css">
   <script   src="https://code.jquery.com/jquery-3.1.0.js"   integrity="sha256-slogkvB1K3VOkzAI8QITxV3VzpOnkeNVsKvtkYLMjfk="   crossorigin="anonymous" defer></script>
+  <script type="text/javascript" src="js/typed.js" defer></script>
```
Comment (dijonkitchen) [inline index.html:12]: Let's put the defers right after script, but before the other attributes. Easier to distinguish.

File: js/typed.js [hunk 4]
```diff
@@ -0,0 +1,438 @@
+// The MIT License (MIT)
```
Comment (dijonkitchen) [inline js/typed.js:1]: I wonder about adding a whole library for a small function, but it looks good!

**Comment by dijonkitchen** [comment]
FYI, there's an empty div (sixth segment, line 63).

**Comment by dijonkitchen** [comment]
Maybe also for the divs with pictures, make them shorter so they don't take up so much screen space.

**Comment by dijonkitchen** [comment]
Mobile looks perfect now! 

For desktop, can we put some left and right padding on the tabs? They're really far out on the side.

**Comment by jhack32** [comment]
👍

**Comment by aattsai** [comment]
I reserved the sixthSegment for additional information that we might want to add... cool quotes, mission, or interesting stats... any suggestions? We can address this on the next PR

**Comment by aattsai** [comment]
Divs with images are already shorter than other divs... I personally like the size now (at least on desktop). Since these images (NYC and SF) serve a purpose of giving people the perception that we are in the heart of the two tech startup hubs. I wanted people to leave with that impression.

**Comment by aattsai** [comment]
I will make more adjustment to the menu bar in the future PRs. Also did you guys play around with the animated underline for the menu bar items?

**Comment by dijonkitchen** [comment]
Yeah, it looks amazing!

**Pull request merged**